### PR TITLE
chore(chromatic): add a command to run chromatic locally

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/5883381568

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
     "storybook": "storybook dev -p 7008",
     "build-storybook": "npm run build:storybook-exports && storybook build",
     "deploy-storybook": "storybook-to-ghpages",
+    "chromatic:local": "chromatic -t $CHROMATIC_PROJECT_TOKEN",
```
Comment (talkor) [inline package.json:78]: Why just `local`? Wouldn't it work if we use if in the action this way?

Comment (YossiSaadi) [reply package.json:78]: is it their default cli command?

Comment (YossiSaadi) [reply package.json:78]: cause using only `chromatic` would override it

Comment (talkor) [reply package.json:78]: ohhh got it

**Pull request merged**